vector/src/main/res/values/strings.xml

```diff
@@ -548,6 +548,8 @@
     <string name="settings_call_category">Calls</string>
     <string name="settings_call_show_confirmation_dialog_title">Prevent accidental call</string>
     <string name="settings_call_show_confirmation_dialog_summary">Ask for confirmation before starting a call</string>
+    <string name="settings_call_ringtone_use_default_stun_title">Allow fallback call assist server</string>
+    <string name="settings_call_ringtone_use_default_stun_summary">Will use %s as assist when your homeserver does not offer one (your IP address will be seen by the stun server during a call)</string>
```

Comment:
`ringtone` is a bit strange here, probably coming from a copy/paste :)

Reply:
> `ringtone` is a bit strange here (for the 2 string keys), probably coming from a copy/paste :)

Yes, I copied the string keys from the old riot-android repo:

https://github.com/vector-im/riot-android/blob/a763655c54c4e465d2ae81e3ecd6a5146564771f/vector/src/main/res/values/strings.xml#L349-L350